tests/test_sheet_collector.py

```diff
@@ -395,3 +397,26 @@ def test_sheet_collector_collect_files_prints_output(tmpdir, test_data, capfd):
     captured = capfd.readouterr()
     for print_item in test_data["collect_files_test"]["expected_print"]:
         assert print_item in captured.out
+
+
+def test_empty_cells_in_sheets_get_type_none():
+    """Check that blank cells are type None."""
+    # input data with empty strings as empty cells
+    input_data = [
+        ["col1", "col2", "col3", "col4"],
+        ["fizz", "", "fooz", "fizz2"],
+        ["buzz", "fooz2", "buzz2", ""],
+    ]
+
+    # expected data with None as empty cells
+    expected_data = [
+        ["col1", "col2", "col3", "col4"],
+        ["fizz", None, "fooz", "fizz2"],
+        ["buzz", "fooz2", "buzz2", None],
+    ]
+
+    # append None where there is a mismatch in data and number of columns/rows
+    new_data = util.fill_to_dimensions(input_data, 4, 3)
+
+    # check to see if empty elements get None
+    assert new_data == expected_data
```

Comment:
Sorry to nit pick on this. I think this test case would fit better in a separate test file that could be something like `test_utils.py`. I don't think it belongs in test_sheet_collector.py`. Can you please create that file and add the test case there instead. 

here is another things that I think would be an improvement. There currently aren't any test cases specifically testing this function. With that in mind, can you check for different kinds of input and outputs that test the whole functionality of this code (aka not just replacing `""` with `None`). 
But besides that I think this PR looks great and we can merge after these changes are added. Thanks for all your work on this!

Reply:
I totally understand, we will get to work on this as soon as we can.